Cubical/Codata/M/AsLimit/M/Base.agda

```diff
@@ -115,30 +115,28 @@ shift-iso S@(A , B) =
         → (p : (n : ℕ) → a (suc n) ≡ a n)
         → (u : (n : ℕ) → B (a n) → Wₙ S n)
         → (n : ℕ)
-        → Iso
-            (PathP (λ x → B (p n x) → Wₙ S n) (πₙ S ∘ u (suc n)) (u n))
-            (πₙ S ∘ (subst (\k -> B k → Wₙ S (suc n)) (α-iso-step-5-Iso-helper0 a p (suc n))) (u (suc n)) ≡ subst (λ k → B k → Wₙ S n) (α-iso-step-5-Iso-helper0 a p n) (u n))
-      α-iso-step-5-Iso-helper1-Iso  a p u n  =
+        → (PathP (λ x → B (p n x) → Wₙ S n) (πₙ S ∘ u (suc n)) (u n)) ≡
```

Comment:
@cmester0 : I simplified shift-iso a little bit. You can probably get rid of all of the other conversion between Iso and path. This should speed up using the proof a lot

Reply:
I think that once you do this you can probably get rid of the lemmas in the helper file. I think most of those are proved in more general form elsewhere in the library now anyway.